azurelinuxagent/common/conf.py

```diff
@@ -626,15 +626,6 @@ def get_normal_upgrade_frequency(conf=__conf__):
     return conf.get_int("Debug.AutoUpdateNormalFrequency", 24 * 60 * 60)
 
 
-def get_enable_ga_versioning(conf=__conf__):
```

Comment:
This is not needed as we have only requested version agent version update path

Reply:
we should still have a "kill switch" for the feature, let's discuss this offline